pyproject.toml

```
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["jupyter-packaging~=0.7.12",
-            "jupyterlab~=3.0.0",
+requires = ["jupyter_packaging~=0.12.2",
+            "jupyterlab==4.*",
```

```suggestion
requires = ["jupyter_packaging>=0.10.6",
            "jupyterlab>=3.6",
```
Those changes are doing away with support for py<3.8, since 3.6 is not supported by jupyter_packaging 0.11+, and jupyterlab 4.0 has removed 3.7.